packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -106,7 +113,7 @@ const unitSelectSizes = ( { selectSize = 'default' }: SelectProps ) => {
 			justify-content: center;
 			align-items: center;
 
-			&:hover {
+			&:not( :disabled ):hover {
 				color: ${ COLORS.ui.borderFocus };
```

Comment:
Looks like this line is redundant? Because we already get the `color` from `baseUnitLabelStyles`.

Reply:
thanks for the review. I didn't get what you mean in this comment.

Reply:
It looks like `color: ${ COLORS.ui.borderFocus }` is unnecessary, because the correct color is already set here:

https://github.com/WordPress/gutenberg/blob/f9d406c5e98b4b4164418a4d19f6905a821c658e/packages/components/src/unit-control/styles/unit-control-styles.ts#L68-L70

It should be ok to remove, unless I'm missing something.

Reply:
Ohh I see. Yes, removed: [`9cd4cec` (#62970)](https://github.com/WordPress/gutenberg/pull/62970/commits/9cd4cecc7df454f18e01b33686d06c67dfff8c13)